firstvoices/backend/admin/characters_admin.py

```diff
@@ -74,37 +111,9 @@ class IgnoredCharacterAdmin(BaseSiteContentAdmin):
 
 
 @admin.register(Alphabet)
-class AlphabetAdmin(BaseSiteContentAdmin):
+class AlphabetAdmin(BaseAdmin):
```

Comment:
Why the change in base class? It's minor but the BaseSiteContentAdmin does a select_related for the site field that helps when there are a lot of items on the list page

Reply:
I changed it (for both) because these models essentially provide 1 object per site, and the only content is a json field. I had updated BaseSiteContentAdmin to let you filter rows per site, and that wasn't necessary when there is only 1 row per site (or less) in the first place. An alternative would be to return this to BaseSiteContentAdmin and remove the filter and duplicate `site` columns, if that would be better.

Reply:
Oh, gotcha! I vote for that alternate way, just to make sure we get all the prefetching everywhere

Reply:
I'm marking this approved because I think it's fine either way. No need to re-review if you make that change :)